_includes/doc.tsx

```diff
@@ -20,7 +20,7 @@ export default function Doc(data: Lume.Data, helpers: Lume.Helpers) {
 
   return (
     <div id="content" className="content">
-      <div class="px-4 sm:px-5 md:px-6 max-w-full">
```

Comment:
Makes the main content go to the middle above ~1200px or so at default font size.